corehq/apps/cleanup/management/commands/evaluate_couch_model_for_sql.py

```diff
@@ -65,7 +68,11 @@ def handle(self, django_app, class_name, **options):
             self.couch_class = to_function(self.models_path)
             self.class_name = self.models_path.split(".")[-1]
 
-        doc_ids = get_doc_ids_by_class(self.couch_class)
+        try:
+            doc_ids = get_doc_ids_by_class(self.couch_class)
+        except HTTPError:
+            view_name = input("Unable to query model. Please enter view to query: ")
+            doc_ids = self.couch_class.get_db().view(view_name).all()
```

Comment:
For repeaters, won't this give all repeaters? I'd think you'd want to add a `doc_type` check, probably at the beginning of `evaluate_doc` since you can't filter within the couch query.